FaceCareRoutine/HomeViewController.swift

```
@@ -1,27 +1,43 @@
 import UIKit
 
-class HomeViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, TipCellDelegate {
+class HomeViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, TipCellDelegate, HomeViewVMDelegate {
     
     let dataStack = DataStack(container: PersistenceService.persistentContainer)
     
     @IBOutlet weak var currentDateLabel: UILabel!
     @IBOutlet weak var daysAmountLabel: UILabel!
     @IBOutlet weak var profileButton: UIButton!
-    
-    
     @IBOutlet weak var cleancePercentLabel: UILabel!
     @IBOutlet weak var moisterizePercentLabel: UILabel!
     @IBOutlet weak var treatPercentLabel: UILabel!
     @IBOutlet weak var protectPercentLabel: UILabel!
-    
-    
     @IBOutlet weak var routinesTableView: UITableView!
     
-    let routines = [Routine(name: "Morning Care"), Routine(name: "Night Care")]
-    var tips = [Tip(title: "Get Some Rest")]
+    public var homeVM: HomeViewVM?
```

Without HomeViewVM our view would not work, hence it should force-unwrap optional. If it fails it should fail early